examples/renderers/NMR/example1.py

```diff
@@ -0,0 +1,121 @@
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Neural Mesh Renderer
+
+# MIT License
+
+# Copyright (c) 2017 Hiroharu Kato
+# Copyright (c) 2018 Nikos Kolotouros
+# A PyTorch implementation of Neural 3D Mesh Renderer (https://github.com/hiroharu-kato/neural_renderer)
+
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+from kaolin.graphics import NeuralMeshRenderer as Renderer
+from kaolin.graphics.nmr.util import get_points_from_angles
+from kaolin.rep import TriangleMesh
+from util import normalize_vertices
+import argparse
+import imageio
+import numpy as np
+import os
+import torch
+import tqdm
+
+ROOT_DIR = os.path.abspath(os.path.dirname(__file__))
+
+###########################
+# Settings
+###########################
+
+CAMERA_DISTANCE = 2.732
+ELEVATION = 30
+TEXTURE_SIZE = 2
+
+
+def parse_arguments():
+    parser = argparse.ArgumentParser(description='NMR Example 1: Render mesh')
+
+    parser.add_argument('--mesh', type=str, default=os.path.join(ROOT_DIR, 'rocket.obj'),
+                        help='Path to the mesh OBJ file')
+    parser.add_argument('--output_path', type=str, default=os.path.join(ROOT_DIR, 'results'),
+                        help='Path to the output directory')
+
+    return parser.parse_args()
+
+
+def main():
+    args = parse_arguments()
+
+    ###########################
+    # Load mesh
+    ###########################
+
+    mesh = TriangleMesh.from_obj(args.mesh)
+    # Normalize into unit cube, and expand such that batch size = 1
+    vertices = normalize_vertices(mesh.vertices).unsqueeze(0).cuda()
```

Comment:
why is mesh.vertices not converted to cuda before being passed to normalize_vertices ?